### PR TITLE
Write read me

### PR DESCRIPTION
**Pull request by jackdavidweber**
In this PR:
- Wrote README.md outlining how to get up and running in our codebase
- Added LICENSE modeled after this [example](https://github.com/google/yapf/blob/master/LICENSE) in another google open source project
- Added CONTRIBUTING.md modeled after this [example](https://github.com/google/earthenterprise/blob/master/CONTRIBUTING.md) in another google open source project
- Documented our [API ](https://app.swaggerhub.com/apis-docs/jackdavidweber/CodeTranslate/1.0.0#/developers/translate)using Swagger

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,7 +1,81 @@
...
+- make sure you are in the root directory
+- create a [virtual enviornment](https://docs.python.org/3/library/venv.html) in python 3
+- copy-paste the following into terminal:
+
+```
+$ pip install -r requirements.txt # for running server
+$ pip install -U pytest # for testing
+$ pip install yapf # for auto formatting
+```
+### Adding service key
```
Comment (g-harel) [inline README.md:34]: Is this optional to run the server?

File: README.md [hunk 2]
```diff
@@ -1,7 +1,81 @@
...
+First and foremost, we want to thank and acknowledge our intern hosts (Danish, Gabriel, and Michael) who guided us and mentored us through every step of our 12 week internship. Thank you for such an amazing experience!
+
+We would also like to acknowledge:
+* Dave Baum for his Level Up Coding lessons
+* Google STEP Intern Program
+* Google SURF Chicago Team 
+
+
+
+
```
Comment (g-harel) [inline README.md:81]: Don't forget to add a LICENSE!

File: README.md (hunk 1 above)
Comment (jackdavidweber) [reply README.md:34]: This was added by @swalsh15 - any thoughts?

File: LICENSE.md [hunk 3]
```diff
@@ -0,0 +1,11 @@
+# Released under MIT License
+
+Copyright (c) 2013 Mark Otto.
+
+Copyright (c) 2017 Andrew Fong.
```
Comment (g-harel) [inline LICENSE.md:5]: 🤔

File: README.md (hunk 1 above)
Comment (swalsh15) [reply README.md:34]: This is optional this was if you wanted to play around with data collection

File: LICENSE.md (hunk 3 above)
Comment (jackdavidweber) [reply LICENSE.md:5]: Good catch haha- who is the owner? Is it Google?

**Comment by g-harel** [comment]
I'd recommend you use [these steps](https://docs.github.com/en/github/building-a-strong-community/adding-a-license-to-a-repository) to generate a license the "standard" way.

File: LICENSE.md (hunk 3 above)
Comment (g-harel) [reply LICENSE.md]: Usually it's the repo owners, you could check out other projects to see what they do.

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+
+We'd love to accept your patches and contributions to this project. There are
+just a few small guidelines you need to follow.
+
+## Contributor License Agreement
+
+Contributions to this project must be accompanied by a Contributor License
+Agreement. You (or your employer) retain the copyright to your contribution,
+this simply gives us permission to use and redistribute your contributions as
+part of the project. Head over to <https://cla.developers.google.com/> to see
```
Comment (g-harel) [inline CONTRIBUTING.md:11]: Where did you find this text? Google generally tries not to associate itself with projects unless they need to.

File: LICENSE [hunk 5]
```diff
@@ -0,0 +1,202 @@
+
+                                 Apache License
```
Comment (g-harel) [inline LICENSE:2]: Why was this changed to MIT?

File: README.md (hunk 1 above)
Comment (g-harel) [reply README.md:34]: I think it would be nice to add a sentence about this only being required for those situations.

File: CONTRIBUTING.md (hunk 4 above)
Comment (jackdavidweber) [reply CONTRIBUTING.md:11]: Found this in other Google Open Source projects. This one specifically came from https://github.com/google/earthenterprise/blob/master/CONTRIBUTING.md

File: LICENSE (hunk 5 above)
Comment (jackdavidweber) [reply LICENSE:2]: Following https://github.com/google/yapf/blob/master/LICENSE in another google open source project

Comment (jackdavidweber) [reply LICENSE]: I ended up switching back when I followed your [comment](https://github.com/jackdavidweber/cjs_capstone/pull/125#issuecomment-667326421) on how to generate license

File: CONTRIBUTING.md (hunk 4 above)
Comment (g-harel) [reply CONTRIBUTING.md:11]: Hmm that looks like an "official" Google open-source project, I don't think this project would require the CLA and there might be some objections to the implication of asking contributors to sign it.
